src/stores/grid_renderer_store.ts

```diff
@@ -607,7 +598,10 @@ export class GridRenderer {
     const showFormula = this.getters.shouldShowFormulas();
     const { x, y, width, height } = this.getters.getRect(zone);
     const { verticalAlign } = this.getters.getCellStyle(position);
-    let style = this.getters.getCellComputedStyle(position);
+    let style = {
+      ...this.getters.getCellComputedStyle(position),
+      ...this.getters.getDataValidationCellStyle(position),
+    };
```

Comment:
I don't like this. Every call to `getCellComputedStyle` is now wrong, except when we add `getDataValidationCellStyle` manually. Data validation style should be included in cell compute style.